tmt/steps/provision/mrack.py

```diff
@@ -866,6 +866,16 @@ class BeakerGuestData(tmt.steps.provision.GuestSshData):
              Submitting user must be a submission delegate for the ``USERNAME``.
              """)
 
+    public_keys: Optional[list[str]] = field(
+        default=None,
+        option='--public-keys',
+        metavar='PUBLICKEYS',
+        help="""
+             If set, Beaker jobs will be submitted on behalf of ``USERNAME``.
+             Submitting user must be a submission delegate for the ``USERNAME``.
+             """,
+        normalize=tmt.utils.normalize_string_list)
```

Comment:
* `public_key` + `public-key` + `multiple=True`. There is already `key`/`--key` key, a singular form, and on the command line I can use it multiple times (`--key foo --key bar --public-key baz ...`)
* `default_factory=list` will be better: it's a list, and an empty list is as good indication of "no keys to install" as `None`, but it's simpler because it's still just a list. So, `list[str]` + `default_factory=list` should work perfectly.
* `help` seems to be copy-pasted & deserves an update.

Reply:
Updated^^ https://github.com/teemtee/tmt/pull/3354/commits/83d8781edb23209858ca8613802b30879941ca5b

Reply:
One more note, I'd recommend tweaking the help text a bit. How about the following?

```
Public keys to add among authorized SSH keys.
```

Reply:
Updated^^:https://github.com/teemtee/tmt/pull/3354/commits/29abf1631e29ae545142901d0d4b3bd6ff488101